portfolio/src/main/java/com/google/sps/data/CommentRequest.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.sps.data.Comment;
+import java.util.List;
+
+
+/** Class containing requested comment data. */ 
+public class CommentRequest {
+
+  /** List of requested comments. */
+  private List<Comment> comments;
+
+  /** Total number of comments. */
+  private int total;
+
+  public CommentRequest(List<Comment> com, int num) {
```

Comment:
It's good to use descriptive variable names when possible.

It took me a bit to learn what 'num' and 'com' were. The screenshots helped!

perhaps 'commentsToReturn' and 'maxNumberOfComments' to return?

After reading through your code, it seems the user can request a limited number of comments and specific comments and the limited number of comments will set a cut off point for the specific comments. I could also see the possibility of an API like this returning all comments asked for, ignoring the max number. It might be worth specifying this behavior in comments.

Reply:
I'll add a better description in CommentRequest about the different behaviors.